src/screens/AnonymousTicketPurchase/ConsequencesScreen.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import {ScrollView, View} from 'react-native';
+import {StyleSheet, useTheme} from '@atb/theme';
+import ThemeText from '@atb/components/text';
+import {useTranslation} from '@atb/translations';
+import AnonymousTicketPurchases from '@atb/translations/screens/subscreens/AnonymousTicketPurchases';
+import {StaticColorByType} from '@atb/theme/colors';
+import {Phone} from '@atb/assets/svg/mono-icons/devices';
+import {Support} from '@atb/assets/svg/mono-icons/actions';
+import {OnboardingStackParams} from '@atb/screens/Onboarding';
+import {MaterialTopTabNavigationProp} from '@react-navigation/material-top-tabs';
+import {RouteProp} from '@react-navigation/native';
+import Actions from '@atb/screens/AnonymousTicketPurchase/components/Actions';
+import Consequence from '@atb/screens/AnonymousTicketPurchase/components/Consequence';
+import FullScreenHeader from '@atb/components/screen-header/full-header';
+
+const themeColor: StaticColorByType<'background'> = 'background_accent_0';
+
+type ConsequencesScreenRouteProps = RouteProp<
+  OnboardingStackParams,
+  'ConsequencesFromOnboarding'
+>;
+
+export type ConsequencesScreenProps = {
+  navigation: MaterialTopTabNavigationProp<OnboardingStackParams>;
+};
+
+const ConsequencesScreen = ({
+  route,
+  navigation,
+}: {
+  route: ConsequencesScreenRouteProps;
+  navigation: ConsequencesScreenProps;
+}) => {
+  const styles = useStyle();
+  const {t} = useTranslation();
+  const {theme} = useTheme();
+
+  const isCallerRouteOnboarding = route?.name === 'ConsequencesFromOnboarding';
+
+  return (
+    <>
+      {!isCallerRouteOnboarding && (
+        <FullScreenHeader leftButton={{type: 'cancel'}} />
+      )}
+      <ScrollView
+        style={styles.container}
+        contentContainerStyle={styles.contentContainer}
+      >
+        <ThemeText
+          type="heading--jumbo"
+          color={themeColor}
+          style={styles.header}
+        >
+          {t(AnonymousTicketPurchases.consequences.title)}
+        </ThemeText>
+        <View>
+          <Consequence
+            value={t(AnonymousTicketPurchases.consequences.messages[0])}
+            icon={
+              <Phone fill={theme.static.background.background_0.background} />
+            }
+          />
+          <Consequence
+            value={t(AnonymousTicketPurchases.consequences.messages[1])}
+            icon={
+              <Phone fill={theme.static.background.background_0.background} />
```

Comment:
icon will be updated after https://github.com/AtB-AS/design-system/pull/58 is merged.